### DISCUSSION TITLE
module not found error when loading from script

### DISCUSSION THREAD

**Discussion by BenedictusAryo** [kedro-org/kedro#895] [Idea]
Hi QB teams,
I just recently discovered kedro and I'm excited to test it to my project

I tried the pandas iris tutorial and having no issue when running from cli 

```
kedro run
```

But, when I create python script inside the project root folder and using the KedroSession as mentioned in: [Lifecycle management with KedroSession](https://kedro.readthedocs.io/en/stable/04_kedro_project_setup/03_session.html)

```
from kedro.framework.session import KedroSession

def main():
    # Do other things

    # Run kedro pipeline
    with KedroSession.create("kedro_iris") as session:
        session.run()


if __name__ == "__main__":
    main()
```
I got ModuleNotFoundError: No module named 'kedro_iris'

here's the asciinema record

<a href="https://asciinema.org/a/436022" target="_blank"><img src="https://asciinema.org/a/436022.svg" /></a>

do I overlooked something?

kedro version:
```
kedro info

 _            _
| | _____  __| |_ __ ___
| |/ / _ \/ _` | '__/ _ \
|   <  __/ (_| | | | (_) |
|_|\_\___|\__,_|_|  \___/
v0.17.5

kedro allows teams to create analytics
projects. It is developed as part of
the Kedro initiative at QuantumBlack.

Installed plugins:
kedro_viz: 3.15.0 (entry points:global,line_magic)
```

Thanks

**Comment by antonymilne** [comment]
Hi @BenedictusAryo,

Looks like the documentation is a bit out of date here. The problem is that the project's `src` directory isn't being add to your python path, which is done in kedro through the `boostrap_project` function. This should work:

```python
from kedro.framework.session import KedroSession
from kedro.framework.startup import bootstrap_project
from pathlib import Path

def main():
    # Do other things

    # Run kedro pipeline
    metadata = bootstrap_project(Path.cwd())
    with KedroSession.create(metadata.package_name) as session:
        session.run()

if __name__ == "__main__":
    main()
```

I'll update the docs accordingly.

**Reply by BenedictusAryo** [reply]
Thanks, @AntonyMilneQB , it works like a charm

**Reply by antonymilne** [reply]
Pleased to hear it, thanks for letting me know. And sorry for the very delayed response - seems like your message was missed in the first place!
